hack/image-cross.sh

```diff
@@ -24,12 +24,12 @@ TAG=${TAG:-$(git describe --tags --always --dirty)}
 
 ARCHES=(amd64 arm64)
 VERSION=v$(cat VERSION)
-QEMUVERSION=7.2.0-1
+QEMUVERSION=7.0.0-28
 TAGS=("$TAG" "$VERSION" latest)
 
 # Build and push the main image
 docker run --rm --privileged \
-    multiarch/qemu-user-static:$QEMUVERSION --reset -p yes
+    tonistiigi/binfmt:qemu-v$QEMUVERSION@sha256:66e11bea77a5ea9d6f0fe79b57cd2b189b5d15b93a2bdb925be22949232e4e55 --install all
```

Comment:
Why are we switching to this docker image?

Reply:
It's just a try. I honestly have no clue why the test fails: https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/post-security-profiles-operator-push-image/1856406308106801152

```
#7 [make 1/1] RUN nix-build nix/default-arm64.nix
#0 0.086 .buildkit_qemu_emulator: /bin/sh: Invalid ELF image for this architecture
#7 ERROR: process "/dev/.buildkit_qemu_emulator /bin/sh -c nix-build $target" did not complete successfully: exit code: 255
------
 > [make 1/1] RUN nix-build nix/default-arm64.nix:
#0 0.086 .buildkit_qemu_emulator: /bin/sh: Invalid ELF image for this architecture
------
Dockerfile
```

The tests started failing 4 weeks ago, and neither buildx nor docker seems to have changed in CI: https://prow.k8s.io/job-history/gs/kubernetes-ci-logs/logs/post-security-profiles-operator-push-image